Tasks/Xcode/Tests/L0.ts

```diff
@@ -589,4 +589,76 @@ describe('Xcode L0 Suite', function () {
             'test result should have been published even when there are test errors');
         done();
     });
+
+    it('macOS auto export', (done: MochaDone) => {
+        this.timeout(1000);
```

Comment:
this won't do anything unless you do `function (done: MochaDone) ...`
https://mochajs.org/#arrow-functions

Reply:
Switched to classic syntax.